tests/android/webview.spec.ts

```diff
@@ -61,3 +62,23 @@ test('should navigate page externally', async function({ androidDevice }) {
   ]);
   expect(await page.title()).toBe('Hello world!');
 });
+
+test('select webview from socketName', async function({ androidDevice }) {
+  test.slow();
+  await androidDevice.shell('am start -n com.android.chrome/com.google.android.apps.chrome.Main about:blank');
+  const webview = await androidDevice.webView({ socketName: 'chrome_devtools_remote' });
```

Comment:
If you did that change above for testing, you can use `isUnderTest()` in the code and use predictable socket name, for example `playwright_test_devtools_remote`